src/frontend/src/handler/create_mv.rs

```diff
@@ -200,35 +236,75 @@ pub async fn handle_create_mv_bound(
     }
 
     let (table, graph, dependencies, resource_group) = {
-        let context = OptimizerContext::from_handler_args(handler_args);
-        let mut with_options = context.with_options().clone();
-
+        let mut with_options = get_with_options(handler_args.clone());
```

Comment:
I think you could simply remove the function and lift the `context` retrieval to this line?

```suggestion
        let context = OptimizerContext::from_handler_args(handler_args);
        let mut with_options = context.with_options().clone();
```

Reply:
Get it! Thanks!